ophyd/areadetector/filestore_mixins.py

```diff
@@ -518,7 +518,7 @@ def stage(self):
 
 
 class FileStoreTIFFSquashing(FileStorePluginBase):
-    '''Write out 'squashed' tiffs
+    r'''Write out 'squashed' tiffs
```

Comment:
[PEP 257](https://www.python.org/dev/peps/pep-0257/) specifies:

> For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. For Unicode docstrings, use u"""Unicode triple-quoted strings""".

But it looks like flake8 doesn't mind triple single quotes (only correctly added the `r`). Not sure if we should care about this? I've seen both ways used throughout various bluesky projects.

Reply:
I did a search for `'''`: there is some number of occurrences of triple single-quotes. I think that they either should be all changed to double quotes (easy) or left alone. What is your opinion?

Reply:
I agree it should be all or nothing, but I'll defer to @danielballan on which it should be.

Reply:
I am not sure, but there may be a reason why `'''` is used in some docstrings in this project.

Reply:
I think that reflects a personal preference of the primary ophyd contributor. I think it would be best to leave them as is in this PR. I wasn't aware that PEP 257 weighed in on this; that's good to know, and it also matches _my_ personal preference. :-)